src/grid/molgrid.py

```diff
@@ -594,17 +594,21 @@ def get_atomic_grid(self, index: int):
             If store=True, the AtomGrid instance used is returned.
             If store=False, the LocalGrid containing points and weights of AtomGrid
             is returned.
+        indices : ndarray(N,)
+            The indices of points corresponding to the atomic grid.
 
         """
         if index < 0:
             raise ValueError(f"index should be non-negative, got {index}")
-        # get atomic grid if stored
+        # get indices of points corresponding to the atomic grid
+        at_indices = np.arange(self._indices[index], self._indices[index + 1])
+        # get atomic grid if stored and the indices within the molecular grid
         if self._atgrids is not None:
-            return self._atgrids[index]
+            return self._atgrids[index], at_indices
         # make a local grid
         pts = self.points[self._indices[index] : self._indices[index + 1]]
         wts = self._atweights[self._indices[index] : self._indices[index + 1]]
-        return LocalGrid(pts, wts, self._atcoords[index])
+        return LocalGrid(pts, wts, self._atcoords[index]), at_indices
```

Comment:
@marco-2023, the `LocalGrid` has an optional argument called `indices` (see https://github.com/theochem/grid/blob/master/src/grid/basegrid.py#L329), so this should return:
`return LocalGrid(pts, wts, self._atcoords[index], indices=at_indices)`

The `AtomGrid` also has `indices` attribute (see https://github.com/theochem/grid/blob/master/src/grid/atomgrid.py#L316), but it is different from `at_indices` returned. Any thought on how to make the two returns consistent?

Reply:
@FarnazH Thank you very much. I will fix the LocalGrid case, regarding the rest, we can discuss it tomorrow. I am not yet clear on the original intended use of the `get_atomic_grid` method. Regarding using it for integrating atomic /local properties inside a molecule, the are several options. We can decide tomorrow how to do it.